### ISSUE TITLE
Node is not defined - Angular5 e2e tests

### ISSUE THREAD

**Issue by UiJo** [dequelabs/axe-core#676]
I use axe-core (3.0.0-alpha.8) together with protractor-accessibility-plugin (0.3.0) to write a11y end to end tests for an Angular5 application. 
protractor.conf.js:
```
exports.config = {
    allScriptsTimeout: 11000,
    plugins: [{
         axe: true,
        treatWarningsAsFailures: true,
        package: 'protractor-accessibility-plugin'
    }],
....
```

The test:
```
 it('has no accessibility violations', function (done) {
                axe.run(function (err, results) {
                    if (err) throw err;
                    expect(results.violations.length).toBe(0);
                    done();
                });
            });
```

When executing the test, in the console I get `Failed: Node is not defined`.
I know that protractor-accessibility-plugin is checking for a11y violation without the need to write an extra test, such as I did, but I want the test to fail when there are violations.

**Comment by dylanb** [comment]
@UiJo the test you have written makes no sense in the context of protractor - you must use the axe-webdriverjs project

**Comment by UiJo** [comment]
@dylanb : The same happens if I do not use protractor-accessibility-plugin

**Comment by marcysutton** [comment]
Sounds like you either need to fetch the `document` with the Protractor API and then run axe inside of the returned promise or it has to be whitelisted with Typescript. Something must have changed since the last time I looked at it.

**Comment by dylanb** [comment]
@UiJo when you run it with the protractor-accessibility-plugin, you are still running it with protractor - right?

In that case you still need to use `axe-webdriverjs`

**Comment by UiJo** [comment]
@dylanb : Yes, I am still using protractor. I don't know how to use axe-webdriverjs in the tests while using  protractor-accessibility-plugin. 
The idea is to make the tests fail when there is a a11y violation. That does not happen by default when using  protractor-accessibility-plugin (is this correct?). I would appreciate any help. Thank you!
